thrift/history.thrift

```diff
@@ -351,6 +351,15 @@ struct DomainFilter {
   20: optional bool reverseMatch
 }
 
+struct GetFailoverInfoByDomainIDRequest {
+  10: optional string domainID
+}
+
+struct GetFailoverInfoByDomainIDResponse {
+  10: optional i32 completedShardCount
+  20: optional list<i32> pendingShards
```

Comment:
Not sure if we have this information either but, would be great to show how many requests remaining per shard for draining the traffic. 

This can be implemented later, not a blocker to this PR

Reply:
yeah. we don't have a good way to measure this. We can think about this later

Reply:
Do you mind creating a task and tracking this in the backlog? We can prioritize it later based on how much we need this information once you ship v0